lib/components/CdeModal.tsx

```diff
@@ -29,23 +32,23 @@ const CdeModal: FC = () => {
         }
     }, [errorMessage, setErrorMessage]);
 
-    const renderStepComponent = (): ReactElement => {
+    const renderStepComponent = (homeStepIndex: number, setHomeStepIndex: Dispatch<SetStateAction<number>>): ReactElement => {
         switch (step) {
             case STEPS.HOME:
-                return <Home/>;
+                return <Home homeStepIndex={homeStepIndex} setHomeStepIndex={setHomeStepIndex}/>;
```

Comment:
Sorry, I didn't understand this point, because homeStepIndex and setHomeStepIndex are not in the context, they are defined in CdeModal.tsx file @afonsobspinto